src/doc/rustc/src/json.md

```diff
@@ -298,6 +298,31 @@ appropriately. (This is needed by Cargo which shares the same dependencies
 across multiple build targets, so it should only report an unused dependency if
 its not used by any of the targets.)
 
+## Timings
+
+**This setting is currently unstable and requires usage of `-Zunstable-options`.**
+
+The `--timings` option will tell `rustc` to emit messages when a certain compilation
+section (such as code generation or linking) begins or ends. The messages will have
+the following format:
```

Comment:
Can you say that the format is unstable and subject to change? I don't want to make *any* promises this early on.